test/unit/controller/QueueControllerSpec.scala

```diff
@@ -103,7 +99,9 @@ class QueueControllerSpec extends UnitSpec with MockitoSugar with WithFakeApplic
       </xml>
       private val request = FakeRequest(POST, "/queue", Headers(CLIENT_ID_HEADER_NAME -> clientId, CONTENT_TYPE -> XML), AnyContentAsEmpty).withXmlBody(xml)
       private val notification = Notification(uuid, Map(CONTENT_TYPE -> XML), xml.toString(), DateTime.now(), None)
+
```

Comment:
minor - breaking up the GIVEN block of code makes it harder to see how the test is structured WRT GIVEN/WHEN/THEN blocks of functionality.

Reply:
fixed